File: backend/src/test/java/org/globe42/test/BaseTest.java
package org.globe42.test;

import org.junit.jupiter.api.BeforeEach;
import org.mockito.MockitoAnnotations;

/**
 * Base class for unit tests, enabling Mockito annotations
 * @author JB Nizet
 */
public class BaseTest {
    @BeforeEach
    public void initMockito() {
        MockitoAnnotations.initMocks(this);
    }
}

File: backend/src/test/kotlin/org/globe42/test/BaseTest.kt
package org.globe42.test

import org.junit.jupiter.api.BeforeEach
import org.mockito.MockitoAnnotations

/**
 * Base class for unit tests, enabling Mockito annotations
 * @author JB Nizet
 */
open class BaseTest {
    @BeforeEach
    fun initMockito() {
        MockitoAnnotations.initMocks(this)
    }
}
